components/BlurhashCanvas.vue

```diff
@@ -25,21 +25,14 @@ export default Vue.extend({
       default: 1
     }
   },
-  watch: {
-    hash() {
-      this.$nextTick(() => {
-        this.draw();
-      });
-    }
-  },
```

Comment:
I'm not sure why we used this in the first place (in fact I don't remember removing it 😋), but everything is working fine on my end without it.